ArjunaJTS/idl/idlj/idlj-tests/pom.xml

```diff
@@ -185,7 +185,7 @@
             <executions>
               <execution>
                 <id>copy-dependencies</id>
-                <phase>initialize</phase>
+                <phase>prepare-package</phase>
```

Comment:
Changed phase because I was getting an error from maven as follows `Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:3.1.1:unpack (copy-dependencies) on project idlj-idl: Artifact has not been packaged yet. When used on reactor artifact, unpack should be executed after packaging: see MDEP-98`